adr/0016-structured-gcse-grades.md

```diff
@@ -0,0 +1,70 @@
+# 16. Structured GCSE grades
+
+Date: 2020-11-18
+
+## Status
+
+Accepted
+
+## Context
+
+Since the service launched in November, candidates have been able to enter details about their academic and other 

```

Comment:
I'd probably just say since the service launched. I think it technically launched in October 2019 (i wasn't actually on the team at the time). I don't think there's any need to specify exactly when though. Looks good other than that 👍